packages/io/stl-serializer/index.js

```diff
@@ -33,11 +33,12 @@ const serialize = (options, ...objects) => {
   objects = utils.flatten(objects)
 
   // convert only 3D geometries
-  objects = objects.filter((object) => geometry.geom3.isA(object))
+  let objects3d = objects.filter((object) => geometry.geom3.isA(object))
 
-  if (objects.length === 0) throw new Error('only 3D geometries can be serialized to STL')
+  if (objects3d.length === 0) throw new Error('only 3D geometries can be serialized to STL')
```

Comment:
Hmm this works, although I wonder if we should throw an error or not ?
I keep wondering if returning Error or Warning objects instead of throwing might be be better for these kind of things (also replacing console.warn etc), this way you always get some useful data out of serializers ??
Meh, let's keep it like you did for now, we can always change later